ludwig/collect.py

```diff
@@ -273,11 +277,7 @@ def cli_collect_activations(sys_argv):
 
     args = parser.parse_args(sys_argv)
 
-    logging.basicConfig(
-        stream=sys.stdout,
-        level=logging_level_registry[args.logging_level],
-        format='%(message)s'
-    )
+    logger.setLevel(logging_level_registry[args.logging_level])
```

Comment:
Do you want to set the log level for the whole package or only for the logger of this file? I think you might want to do something like: `getLogger("luwdig").setLevel(...)`

Reply:
yeah, you're right. Will update.